src/content/en/updates/2017/11/overscroll-behavior.md

```diff
@@ -0,0 +1,370 @@
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+{% include "web/_shared/contributors/majidvp.html" %}
+{% include "web/_shared/contributors/sunyunjia.html" %}
+
+<style>
+figure {
+  text-align: center;
+}
+figcaption {
+  font-size: 14px;
+  font-style: italic;
+}
+.flex {
+  display: flex;
+}
+.flex1 {
+  flex: 1;
+}
+.border {
+  border: 1px solid #ccc;
+}
+</style>
+
+### TL;DR {: #tldr .hide-from-toc}
+
+The [CSS `overscroll-behavior`][spec] property allows developers to override the
+browser's default overflow scroll behavior when reaching the top/bottom of
+content. Examples use cases include disabling the browser's pull-to-refresh
+feature on mobile, removing overscroll glow and rubberbanding effects,
+and preventing page content from scrolling when it's beneath a modal/overlay.
+
+CSS `overscroll-behavior` is requires Chrome 63+ and is development or being
+considered by other browsers. See
+[chromestatus.com](https://www.chromestatus.com/feature/5734614437986304) for
+more information.
+{: .caution }
+
+## Background
+
+### Scroll boundaries and scroll chaining {: #scrollchaining }
+
+<figure class="attempt-right">
+  <a href="/web/updates/images/2017/11/overscroll-behavior/drawer-scroll.mp4"
+     target="_blank">
+    <video src="/web/updates/images/2017/11/overscroll-behavior/drawer-scroll.mp4"
+           autoplay loop alt="Drawer demo" height="300"></video>
+  </a>
+  <figcaption>Scroll chaining on Chrome Android.</figcaption>
+</figure>
+
+Scrolling is one of the most fundamental ways to interact with a page, but 
+certain UX patterns can be tricky to deal with because of the browser's quirky 
+default behaviors. As an example, take an app drawer with a large number of 
+items that the user may have to scroll through. When they reach the bottom, the
+overflow container stops scrolling because there's no more content to consume. 
+In other words, the user
+reaches a "scroll boundary". But notice what happens if the user continues to 
+scroll. **The content *behind* the drawer starts scrolling**! Scrolling is 
+taken over by the parent container; the main page itself in the example. 
+
+Turns out this behavior is called  **scroll chaining**; the browser's default 
+behavior when scrolling content. Oftentimes the default is pretty nice, but 
+sometimes it's not desirable or even unexpected. Certain apps may want to 
+provide a different user experience when the user hits a scroll boundary.
+
+### The pull-to-refresh effect {: #p2r }
+
+Pull-to-refresh is a intuitive gesture popularized by mobile apps such as
+Facebook and Twitter. Pulling down on a social feed and releasing creates new 
+space for more recent posts to be loaded. In fact, this particular UX has 
+become _so popular_ that mobile browsers like Chrome on Android have adopted 
+the same effect. Swiping down at the top of the page refreshes the entire page:
+
+<div class="flex">
```

Comment:
are you trying to do two side by side videos here? if so, you can use `attempt-left` and `attempt-right` I can't check since the PR is coming from your own personal repo

Reply:
done